test/performance/parse-performance-metrics.R

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env Rscript
```

Comment:
I am not familiar with r-script.  Did you pull this from somewhere else?  Should we be vendoring this into the project or can it be sourced from elsewhere?

Reply:
Oh, this is my magic, I used to work for R :)